sources/Interop/Advapi32/um/wincrypt/Advapi32.cs

```diff
@@ -0,0 +1,1082 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/wincrypt.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public static unsafe partial class Advapi32
+    {
+        private const string LibraryPath = "advapi32";
+
+        [DllImport(LibraryPath, CallingConvention = CallingConvention.Winapi, EntryPoint = "CryptAcquireContextA", ExactSpelling = true)]
+        [return: NativeTypeName("BOOL")]
+        public static extern int CryptAcquireContextA([NativeTypeName("HCRYPTPROV *")] uint* phProv, [NativeTypeName("LPCSTR")] sbyte* szContainer, [NativeTypeName("LPCSTR")] sbyte* szProvider, [NativeTypeName("DWORD")] uint dwProvType, [NativeTypeName("DWORD")] uint dwFlags);
```

Comment:
Several of these, like `HCRYPTPROV` are handle types that don't go through the same define as other handles like `HWND`

You should be able to get most of them by remapping `ULONG_PTR=UIntPtr` and `LONG_PTR=IntPtr`

Reply:
Fixed.

From my research it seems like only `ULONG_PTR` was required and only for `wincrypt.h`.